src/app/examples/todos/todos.actions.spec.ts

```diff
@@ -4,11 +4,8 @@ describe('Todos Actions', () => {
   describe('TodosAdd', () => {
     it('should create an action', () => {
       const action = new fromTodos.ActionTodosAdd({ name: 'test' });
-
-      expect({ ...action }).toEqual({
-        type: fromTodos.TodosActionTypes.ADD,
-        payload: { name: 'test' }
-      });
+      expect(action.type).toEqual(fromTodos.TodosActionTypes.ADD);
```

Comment:
Personally I like to use `jasmine.objectContaining` for these tests.
We also want to test if the `id` property is defined.

This together would become something like:
```ts
      expect({ ...action }).toEqual(
        {
          type: fromTodos.TodosActionTypes.ADD,
          payload: jasmine.objectContaining({ name: 'test' })
        }
      );
      expect(action.payload.id).toBeDefined();
```